snap/snapcraft.yaml

```diff
@@ -0,0 +1,148 @@
+# yamllint disable rule:line-length
```

Comment:
Necessary for charmed PostgreSQL only?

Reply:
Linting passes without it, not sure why the rule's disabled and not extended to longer length.